packages/labs/analyzer/src/lib/lit-element/lit-element.ts

```diff
@@ -33,18 +39,75 @@ export const getLitElementDeclaration = (
   declaration: LitClassDeclaration,
   analyzer: AnalyzerInterface
 ): LitElementDeclaration => {
+  const reactiveProperties = getProperties(declaration, analyzer);
   return new LitElementDeclaration({
     tagname: getTagName(analyzer.typescript, declaration),
     // TODO(kschaaf): support anonymous class expressions when assigned to a const
     name: declaration.name?.text ?? '',
     node: declaration,
-    reactiveProperties: getProperties(declaration, analyzer),
+    reactiveProperties,
     ...getJSDocData(declaration, analyzer),
     getHeritage: () => getHeritage(declaration, analyzer),
-    ...getClassMembers(declaration, analyzer),
+    ...getLitElementClassMembers(declaration, analyzer, reactiveProperties),
   });
 };
 
+function getCustomElementField(
+  field: ClassField,
+  prop: ReactiveProperty
+): CustomElementField {
+  const {name} = field;
+  const {
+    privacy,
+    inheritedFrom,
+    source,
+    readonly,
+    type,
+    description,
+    deprecated,
+    summary,
+  } = field;
```

Comment:
why two destructuring statements here? combine?

Reply:
removed the destructuring in favour of property assignment